client/gutenberg/extensions/tiled-gallery/layout/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import photon from 'photon';
+import { __, sprintf } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Square from './square';
+import GalleryImage from '../gallery-image';
+
+export default class Layout extends Component {
+	photonize( { height, width, url } ) {
+		const { layoutStyle } = this.props;
+		if ( [ 'circle', 'square' ].includes( layoutStyle ) && width && height ) {
+			const size = Math.min( width, height );
+			return photon( url, { resize: `${ size },${ size }` } );
+		}
+		return photon( url );
+	}
+
+	renderImage( img, i ) {
+		const {
+			columns,
+			imageCrop,
+			images,
+			linkTo,
+			onRemoveImage,
+			onSelectImage,
+			selectedImage,
+			setImageAttributes,
+		} = this.props;
+
+		/* translators: %1$d is the order number of the image, %2$d is the total number of images. */
+		const ariaLabel = __( sprintf( 'image %1$d of %2$d in gallery', i + 1, images.length ) );
```

Comment:
`__` and `sprintf` need to be another way around here so that we don't attempt to translate already sprintf'ied result.

Reply:
This is straight from Gutenberg, so they would have this issue as well:

https://github.com/WordPress/gutenberg/blob/5dba988cf740653b9ab41672c462c785f1ff1a38/packages/block-library/src/gallery/edit.js#L267-L268